src/ui/windows/TogglDesktop/TogglDesktop/ui/views/TimerEntryListView.xaml

```diff
@@ -4,20 +4,93 @@
              xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
              xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
              xmlns:toggl="clr-namespace:TogglDesktop"
+             xmlns:vm="clr-namespace:TogglDesktop.ViewModels"
              mc:Ignorable="d" 
-             d:DesignHeight="300" d:DesignWidth="400">
+             d:DesignHeight="300" d:DesignWidth="400"
+             d:DataContext="{d:DesignInstance vm:TimerEntryListViewViewModel, IsDesignTimeCreatable=False}">
     <Grid>
         <Grid.RowDefinitions>
             <RowDefinition Height="Auto"/>
             <RowDefinition />
         </Grid.RowDefinitions>
 
-        <toggl:TimeEntryList Grid.Row="1"
-                             x:Name="Entries"
-                             FocusTimer="onFocusTimer"/>
+        <TabControl Grid.Row="1" Name="TabControl" SelectedIndex="{Binding SelectedTab}">

```

Comment:
Personally I think the actual TimeEntryListView layout will be more readable if the styles for TabItem and TabControl are moved into the resources, even if they are used only once. But I don't have a strong opinion on this. What's the benefit of putting the styles together with the view?

Reply:
We already have a TabControl style in Resources, and it is automatically applied to all tabcontrols in the app. It is used in Preferences, but I was not able to use this style for this case as I needed slightly different behavior. I can create one more TimeEntryTabControl in Resources and with the `x:Key` property, but then I'll have to specify the name of the style for each tab (I know, there are only 2 tabs, but still it's more convenient to have a style applied to all tabs automatically).